packages/manager/apps/dedicated/client/app/css/less/dedicated/server/installation.less

```diff
@@ -105,3 +105,19 @@
     color: @red_const;
   }
 }
+
+.form-control.oui-input_l,
+.form-control.oui-input_s {
+  width: inherit;
+}
+
+.fix-help-block .help-block {
+  // this is to compensate the move down of the next questions when an error is displayed
+  margin-bottom: -25px;
+  height: 20px;
+}
+
+.fix-help-block .oui-field__footer {
+  // this is to hide the native lib errors because they are customized
+  display: none;
```

Comment:
If you have a better idea to hide the lib native error message (because I use custom error messages) I will be very happy

Reply:
The rules to display errors in a oui-field are the following:

- a form is encapsulating the field
- the input is filled or it has lost focus or the form has been submitted

So the only way (other than using css) to not diplay it is to remove the encapsulating form (which may not be possible when using wizard)

Your solution seems OK to me